bokeh/palettes.py

```diff
@@ -144,6 +142,10 @@
 
 :Viridis256: :bokeh-palette:`viridis(256)` (mpl)
 
+
+Additional, several 256-color perceptually uniform palettes are
```

Comment:
Additionally

Reply:
Oops; moved that text without reading it closely enough! :-)  Thanks.